projects/eslint-plugin/rules/strict-tui-doc-example.js

```diff
@@ -0,0 +1,95 @@
+const getTypeName = require('../utils/get-type-name');
+
+const MESSAGE_ID = `strict-doc-example-extensions`;
+const ERROR_MESSAGE = `The import path extension should match the object key`;
+
+const DOC_EXAMPLE_INTERFACE_NAME = 'TuiDocExample';
+
+/**
+ * @type {TuiDocExample}
+ */
+const fileNameToExtension = {
+    TypeScript: '.ts',
+    HTML: '.html',
+    LESS: '.less',
+};
+
+/**
+ * Parses path into RegExp groups.
+ * @param {string} path
+ * @returns {RegExpMatchArray | null}
+ */
+const getPathGroups = path => {
```

Comment:
const getPathGroups = path => path.match(/(.+)(\.(?:ts|less|scss|js|html))(.*)/);

Reply:
are you intentionally removed the backslash? 

![image](https://user-images.githubusercontent.com/22116465/215325056-b7afaaf9-2d46-4c3d-a6b2-
9a9abada4f87.png)

Reply:
sorry, maybe github escaped some characters